src/app.component.ts

```diff
@@ -4,10 +4,22 @@ import { AboutPageComponent } from './aboutpage/aboutpage.component';
 import { FooterComponent } from './appfooter/leveragefooter.component';
 import { HeaderComponent } from './appheader/leverageheader.component';
 
+import { Router, RouterModule } from '@angular/router';
+
+interface Nav {
```

Comment:
This interface should be moved to the models folder and imported into this component